backend/business-logic/users.js

```diff
@@ -73,20 +72,17 @@ const userManager = {
       throw new Error(`Could not find such a user!`);
     }
   },
-  getAllUsers: async () => {
-    const usersAll = await userStore.all();
-    console.log(`GETALLuserS: ${usersAll}`);
-    if (!usersAll) {
-      throw new Error(`Could not find any users!`);
+  getAllVolunteers: async () => {
+    try {
+      const volunteers = await User.find({ userType: "volunteer" });
+
+      if (User.userType !== volunteers) {
+        throw new Error(`Could not find any volunteer`);
+      }
+    } catch (err) {
+      console.log(err.message);
     }
-    return usersAll;
-  },
-  putUser: async (user) => {
-    return userStore.update(user.id, user);
-  },
-  deleteUser: async (userId) => {
-    await userStore.remove(userId);
-    return true;
+    return volunteers;
```

Comment:
just make sure that this is actually part of the try part of the function :) and then it works :)
so in line 78..(and you dont need the if statement..)